ouroboros-consensus/docs/HardFork.md

```diff
@@ -0,0 +1,211 @@
+# Details of the hard fork transition
+
+This document attempts to describe the details of the hard fork transition
+from Byron to Shelley, and from Shelley to future versions of the ledger.
+
+## Byron
+
+The Byron specification can be found at
+https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/byronLedgerSpec/latest/download-by-type/doc-pdf/ledger-spec .
```

Comment:
```suggestion
The Byron specification can be found [here](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/byronLedgerSpec/latest/download-by-type/doc-pdf/ledger-spec).
```

Reply:
Tssh. Tim Berners-Lee would wince if he saw it :roll_eyes: :grimacing: